### PR TITLE
Added quotes to monkey type

### PR DESCRIPTION
**Pull request by TheOrdinaryWizard**
### Description

Added a Pulp Fiction quote to english.json

### CODE REVIEW

File: frontend/static/quotes/english.json [hunk 1]
```diff
@@ -38807,5 +38807,11 @@
       "id": 6762,
       "length": 152
     }
```
Comment (rizwanmustafa) [inline frontend/static/quotes/english.json:38809]: Add a comma at the end of this line

**Review by rizwanmustafa — CHANGES_REQUESTED** [review]

File: frontend/static/quotes/english.json [hunk 2]
```diff
@@ -38807,5 +38807,11 @@
...
+    {
+      "text": "Aw, man, I shot Marvin in the face.",
+      "source": "Pulp Fiction",
+      "id": 6762,
```
Comment (lachie-underhill) [inline frontend/static/quotes/english.json:38813]: This id should be incremented to 6763

**Comment by rizwanmustafa** [comment]
Closing this PR due to inactivity.

**Comment by monkeytypegeorge** [comment]
Continuous integration check(s) failed. Please review the failing check's logs and make the necessary changes. https://github.com/monkeytypegame/monkeytype/actions/runs/4029666377

**Comment by Bruception** [comment]
@TheOrdinaryWizard , sorry for the spam. Your PR can RIP now

**Pull request closed without merging**